src/myBestClassEver.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace src\Controller;
+
+class myBestClassEver
+{
+    protected function myStringContainsToto(string $stringToTest)
+    {
+        $containsToto = false;
+
+        if (strpos($stringToTest, 'toto') == false) {
```

Comment:
yoda condition ici + triple =
```suggestion
        if (false === strpos($stringToTest, 'toto')) {
```

Reply:
On pourrait direct renvoyer strpos($stringToTest, 'toto')
la variable $containsToto ne sert à rien en soit
et d'ailleurs le true/false est inversé ici non ? actuellement on renvoie true si strpos === false donc si on trouve pas toto 🤔